mewarpx/tests/test_diagnostic_plotting.py

```diff
@@ -0,0 +1,212 @@
+"""
+Test that verifies data is plotted correctly from the
+Monte-Carlo Collision script based on case 1 from
+Turner et al. (2013) - https://doi.org/10.1063/1.4775084
+"""
+import pytest
+import os.path
+
+from mewarpx import util as mwxutil
+
+from pywarpx import picmi
+
+import numpy as np
+
+constants = picmi.constants
+
+
+@pytest.mark.parametrize("plot_on_diag_steps, post_processing",
+[
+    (True, False),
+    (False, True)
```

Comment:
I guess with this set you could just use one and set the other one to be NOT that.

Reply:
Using `not` on tuples or lists checks if they're empty instead of negating all the values. I could change these to lists and do:
```Python
    [True, False],
    [not e for e in [True, False]]
```
But the current way seems cleaner.

Reply:
What I meant is that you can just use 1 input for the test, say `plot_on_diag_steps` and if it is false you do `post_processing`. Does that make sense?